.env.development

```diff
@@ -1,5 +0,0 @@
-RUST_LOG=warn,info,error,debug
```

Comment:
I think that we should keep this file. It was working by default, now we need to create a .env file to make this works.

Reply:
Thing is, not everybody is going to use the same environment, some people may want to use different values for such environment variables.

The most common pattern is to have a `.env.sample` which provides predefined values, but the project will consume a copy of that file under the name `.env` which is ignored by Git and every collaborator will keep a copy of that file locally.